code/game/objects/items/storage/firstaid.dm

```diff
@@ -339,6 +340,22 @@
 	. = ..()
 	update_icon()
 
+/obj/item/storage/pill_bottle/refill(mob/user)
+	. = ..()
+	if(!.)
+		return
+	var/available_space = max_storage_space
+	for(var/obj/item/stored_item in contents)
```

Comment:
It's sort of weird to leave the existing things in when you're essentially refreshing the item otherwise, but I guess it's better?